tests/integrationTests/tests/c_failure_messages/__init__.py

```diff
@@ -37,8 +38,12 @@ def cleanup(test):
 def schema_validation(test):
     cleanup(test)
     config_path = os.path.join(test.testcase_path, 'assignment_config', 'complete_config.json')
-    test.validate_complete_config(config_path)
-   
+    try:
+        test.validate_complete_config(config_path)
+    except Exception:
+        traceback.print_exc()
+        raise
```

Comment:
I think we should modify this block:
https://github.com/Submitty/Submitty/blob/master/tests/integrationTests/lib.py#L118-L128

to print out the full exception stack trace (instead of just the error message) instead of having this pattern in every schema validation testcase.

Reply:
Resolved per conversation at today's meeting.